package.json

```diff
@@ -32,7 +32,7 @@
     "express-session": "^1.16.1",
     "git-branch": "^2.0.1",
     "gulp": "^3.9.1",
-    "gulp-autoprefixer": "^4.0.0",
+    "gulp-autoprefixer": "4.1.0",
```

Comment:
so we are incrementing the version number?

Can we change one of the date fields in a page to type=number to confirm it is working?